README.md

```diff
@@ -1,7 +1,8 @@
 # Horton
-A repository for doing shared R&amp;D on Sufia for the Development team.
+A repository for doing shared R&amp;D on Hyrax for the Development team.
```

Comment:
Can we tweak this wording a bit? Something like:

A digital collections application for the UC San Diego Library built using the [Hydra framework](https://projecthydra.org/), specifically using the [Hyrax front-end](https://github.com/projecthydra-labs/hyrax/), and the [Fedora Repository](http://fedorarepository.org/).